test-e2e/k8s.py

```diff
@@ -103,9 +103,18 @@ def pod_exec(self, name, cmd):
                       stderr=True, stdin=False,
                       stdout=True, tty=False,
                       _preload_content=False)
-
         while resp.is_open():
             resp.update(timeout=1)
             err = resp.read_channel(ERROR_CHANNEL)
             if err:
-                return yaml.load(err)
\ No newline at end of file
+                return yaml.load(err)
+
+    def pod_exec2(self, name, cmd):
```

Comment:
I assume you have this because you don't need to examine the output, rename it to pod_exec_detached